src/scripts/cmake/Options.cmake

```diff
@@ -22,6 +22,15 @@ option(BUILD_STATIC "Build ESBMC in static mode (default: OFF)" OFF)
 option(BUILD_DOC "Build ESBMC documentation" OFF)
 option(ENABLE_REGRESSION "Add Regressions Tests (default: OFF)" OFF)
 
+#############################
+# SOLVERS
+#############################
+option(ENABLE_BOOLECTOR "Use Boolector solver (default: OFF)" OFF)
+option(ENABLE_Z3 "Use Z3 solver (default: OFF)" OFF)
+option(ENABLE_MATHSAT "Use MathSAT solver (default: OFF)" OFF)
+option(ENABLE_YICES "Use Yices solver (default: OFF)" OFF)
+option(ENABLE_CVC4 "Use CVC4 solver (default: OFF)" OFF)
+
```

Comment:
Does this mean that I'll have to enable the solvers when I configure the cmake build? Or will it work as it works today, that it's enabled if we have the solvers in the $SATDIR64 or $HOME directory?

Reply:
> Does this mean that I'll have to enable the solvers when I configure the cmake build?

If you want to use the PATH version yes. If not, only defining the directory will work.

> Or will it work as it works today, that it's enabled if we have the solvers in the $SATDIR64 or $HOME directory?

No, I forgot about this.

Reply:
I think support only $HOME is fine.

Reply:
It is checking for $HOME directory now.